src/app/hooks/use-profile-synchronizer.ts

```diff
@@ -332,14 +340,38 @@ export const useProfileSynchronizer = ({ onProfileRestoreError }: ProfileSynchro
 
 			if (shouldSync()) {
 				setStatus("syncing");
+				onProfileSyncStart?.();
 
-				await profile.sync();
-				await persist();
+				try {
+					await profile.sync();
+					await persist();
+					runAll();
+
+					// for better performance no need to await
```

Comment:
```suggestion
					// for better performance no need to await
                  runAll();
```

i think this is a comment for `runAll()`

Reply:
Thanks @w3ea